### PR TITLE
Update packages and use webpack 4

### PR DESCRIPTION
**Pull request by karelhala**
All packages updated, webpack config changed to follow v4. No UI changes or anything, just build.

https://5b16ab7db3127433dd13bbab--sleepy-spence-62a12c.netlify.com

### CODE REVIEW

**Comment by priley86** [comment]
i like the approach w/ netlify b.t.w. ;) - i think we will be using the same path in PF React soon...

**Comment by Hyperkid123** [comment]
@karelhala just one question. Why do we want alpha packages? 

Also, storybooks os netlify are broken. Something is requesting jQuery `Uncaught Error: Cannot find module 'jquery'`

**Comment by priley86** [comment]
this will unblock our QE team on testing latest v2v UI. It's much appreciated if we can get this in soon...
https://github.com/ManageIQ/miq_v2v_ui_plugin/issues/363

cc: @bthurber @Yadnyawalkya

**Comment by AparnaKarve** [comment]
@karelhala How do you suggest we test this locally?

Will this work in classic-ui package.json  - 
```
"@manageiq/react-ui-components": "https://github.com/karelhala/react-ui-components#update/pf-react-v2",
```

File: package.json [hunk 1]
```diff
@@ -49,67 +49,69 @@
...
+    "style-loader": "^0.21.0",
+    "url-loader": "^1.0.1",
+    "webpack": "^4.10.2",
+    "webpack-cli": "^3.0.2",
+    "webpack-dev-server": "^3.1.4"
   },
   "dependencies": {
-    "final-form": "^4.6.1",
-    "patternfly-react": "^1.16.4",
+    "extract-text-webpack-plugin": "^4.0.0-beta.0",
```
Comment (himdel) [inline package.json]: Was this one intentional?

Comment (karelhala) [reply package.json]: I tried it, but should be removed, good catch!

**Comment by karelhala** [comment]
@Hyperkid123:
> just one question. Why do we want alpha packages?

Webpack is improving way too fast compared to storybook, but they released alpha version.

@AparnaKarve:
> Will this work in classic-ui package.json -

I don't think that it's going to work. Since we publish build into npm and not into github.

**Comment by karelhala** [comment]
@Hyperkid123 great catch! Removing jquery from provide plugins works and netlify does not complain about it anymore.

**Comment by AparnaKarve** [comment]
> I don't think that it's going to work. Since we publish build into npm and not into github. If you want to test it you'd have to download this package and build it locally.

@karelhala Thanks. I will just wait until it's merged... hopefully today :)

File: scripts/storybook/webpack.config.js [hunk 2]
```diff
@@ -15,18 +15,34 @@ module.exports = {
...
+              },
             },
-          ],
-        }),
+          },
+        ],
+        enforce: 'pre',
+      },
+      {
+        test: /\.s?[ac]ss$/,
```
Comment (himdel) [inline scripts/storybook/webpack.config.js]: Not sure if `/\.(sa|sc|c)ss$/` wouldn't be safer - not sure any of those loaders have `.ass` file support.

Comment (karelhala) [reply scripts/storybook/webpack.config.js]: will change it

**Comment by karelhala** [comment]
@himdel Provide plugin kinda broke our builds on netlify because it exposed jquery and other variables, which is unwanted. And we didn't really used it anyway.

**Pull request merged**